kong/runloop/plugin_servers/init.lua

```diff
@@ -153,7 +168,7 @@ function get_instance_id(plugin_name, conf)
 
   while instance_info and not instance_info.id do
     -- some other thread is already starting an instance
-    ngx.sleep(0)
+    ngx_sleep(0)
```

Comment:
Could we use `native_ngx_sleep` here?

See PR #9523

Reply:
What about using `kong.tools.utils.yield`?

Reply:
I am not familiar with the plugin server, not sure about the logic of it, let's do it later.